assets/src/js/admin/admin-setup.js

```diff
@@ -8,12 +8,12 @@
  */
 
 Array.from( document.querySelectorAll( '.setup-item' ) ).forEach( ( setupItem ) => {
-	const actionAnchor = setupItem.querySelector( '.action a' );
+	const actionAnchor = setupItem.querySelector( '.js-action-link' );
 
 	if ( actionAnchor ) {
 		actionAnchor.addEventListener( 'click', ( e ) => e.stopPropagation() );
 		setupItem.style.cursor = 'pointer';
-		setupItem.addEventListener( 'click', ( event ) => {
+		setupItem.addEventListener( 'click', ( event ) => { // eslint-disable-line no-unused-vars
```

Comment:
The linter kept adding the `event` parameter to the callback and would throw an error because it wasn't used, so I disabled the rule on the line.